auth/Program.cs

```diff
@@ -0,0 +1,140 @@
+﻿/*
+ * Copyright (c) 2015 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+// [START all]
+
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Storage.v1;
+using Google.Apis.Storage.v1.Data;
+using Google.Apis.Services;
+using System;
+using System.Threading.Tasks;
+
+namespace GoogleCloudSamples
+{
+    public class AuthSample
+    {
+        private const string usage = @"Usage:AuthSample <bucket_name>";
+        // [START build_service]
+        /// <summary>
+        /// Creates an authorized Cloud Storage client service using Application 
+        /// Default Credentials.
+        /// </summary>
+        /// <returns>an authorized Cloud Storage client.</returns>
+        public static async Task<StorageService> CreateAuthorizedClientAsync()

```

Comment:
`CodeFormatter.exe` adds explicit visibility keywords.  Is there a `/rule:` option to *not* do this?